### PR TITLE
Add Lonely Star to the newsletter

### PR DESCRIPTION
**Pull request by 17cupsofcoffee**
As discussed on Discord with Ozkriff, I didn't feel like there was enough Tetra updates to justify a seperate section, so I bundled it together :smile:

Rendered:

image

### CODE REVIEW

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -76,6 +76,28 @@ Features:
...
 
+### [Lonely Star][lonely-star]
+
+> *Space is a lonely place, but at least you've got the music to keep you company.*
+
+[Lonely Star][lonely-star] is a side-scrolling infinite runner, with simple generative music. You collect orbs and fly through rings in order to play notes and stay alive.
+
+It was developed by [17cupsofcoffee][17cupsofcoffee], using the [Tetra][tetra] 2D game framework, for [Weekly Game Jam #135](weekly-game-jam-135).
+
+[![Lonely Star screenshot](lonely-star.PNG)][lonely-star]
```
Comment (ozkriff) [inline content/posts/newsletter-007/index.md]:
```suggestion
[![Lonely Star screenshot](lonely-star.png)][lonely-star]
```

File: content/posts/newsletter-007/index.md [hunk 2]
```diff
@@ -76,6 +76,28 @@ Features:
...
+A
```
Comment (ozkriff) [inline content/posts/newsletter-007/index.md]: Also, this lonely A looks a little bit weird)

File: content/posts/newsletter-007/index.md (hunk 1 above)
Comment (17cupsofcoffee) [reply content/posts/newsletter-007/index.md]: Whoops, copied and pasted from the other screenshot, my bad!

File: content/posts/newsletter-007/index.md (hunk 2 above)
Comment (17cupsofcoffee) [reply content/posts/newsletter-007/index.md]: Agh, I think I mashed my keyboard by mistake when committing 😆 Will fix.

Comment (17cupsofcoffee) [reply content/posts/newsletter-007/index.md]: Fixed, and rebased on top of the other PR that just merged,

**Pull request merged**